testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -1467,17 +1479,22 @@ public void getRows() throws IOException {
     assertThat(data.get(0).get("firstName")).isEqualTo("John");
   }
 
+  // Test for inserting and fetching row(s) with Tuple values: inserts using
+  // "Stringified" (non-JSON, CQL literal) notation.
   @Test
-  public void getRowsWithTuple() throws IOException {
+  public void getRowsWithTupleStringified() throws IOException {
     createKeyspace(keyspaceName);
     createTestTable(
         tableName,
-        Arrays.asList("id text", "data tuple<int,boolean>"),
+        Arrays.asList("id text", "data tuple<int,boolean,text>", "alt_id uuid"),
         Collections.singletonList("id"),
         Collections.emptyList());
+    String altUid1 = UUID.randomUUID().toString();
     insertTestTableRows(
         Arrays.asList(
-            Arrays.asList("id 1", "data (28,false)"), Arrays.asList("id 2", "data (39,true)")));
+            // Put UUID for the first row; leave second one empty/missing
+            Arrays.asList("id 1", "data (28,false,'foobar')", "alt_id " + altUid1),
+            Arrays.asList("id 2", "data (39,true,'bingo')")));
```

Comment:
Will NOT yet attempt explicit null, since Stringified handler will not accept it. But whether it should or not is open to further discussion -- there is no easy way to tell apart String value "null" from something that we'd like to be `null`.

So for now fix is strictly for JSON payload case.